install_files/ansible-base/securedrop-logs.yml

```diff
@@ -94,7 +94,7 @@
         --batch --yes
         "{{ log_tarball_filename }}"
       environment:
-        GNUPG_HOME: /home/amensia/.gnupg
+        GNUPG_HOME: /home/amnesia/.gnupg
```

Comment:
Scary.

Reply:
Very quick review tells me that since we are fetching they key with the previous action, it should not be a security fail, but I would like to review that logic anyway. Why are we using amnesia to begin with?

Reply:
FWIW, amnesia is the default Tails user.